recipes/libcflib/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libcflib" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa425a47b23b2beac5888402a74f8ccec611f8e47f3a7930215427ccc732ea92
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cerberus
+    - tornado
+    - whoosh
+    - xonsh
+    - zict
+    - tqdm
+    - toolz
+    - jinja2
+    - lazyasd
+    - pandas
+
+test:
+  imports:
+    - libcflib
+    - libcflib.environ
+
+about:
+  home: http://github.com/regro/libcflib
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Library Conda Forge Library '
+
+extra:
+  recipe-maintainers:
+    - scopatz
+    - CJ-Wright
+    - mariusvniekerk
```

Comment:
@CJ-Wright @mariusvniekerk - are you OK with being maintainers?

Reply:
Yes of course! :smile:

Reply:
Sure